css-positioning/fixed.css

```diff
@@ -0,0 +1,10 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  right: 300px;
```

Comment:
😱 😱 😱 😱 

This is a _**big movement**_. Is this to center the text? If so, you should just make the element the full width of its containing block, it already has `text-align: center` applied to it.